.travis.yml

```diff
@@ -15,6 +15,5 @@ deploy:
   api_key:
     secure: qofM3BtlKhNbnirVWpHS15WPXDWH88MNEbo8AyVM+Y08AvzDUdbgbHQxtPn07Ke1K0X2ZexqNLV6TDbatvIW3nXxPm23DtSySb9DVpegpW/TD+C0ryO4emta3eaRmOIbnt+Aq8XSKnOBr6BOchW9EKJnFmCexJqiq935SarELtmU+J3LncEwZQuTQyVs8UpwBFtOryBkugFwRVhMjpHYqChsu+bgsDToMbjCZp8YD5e9GHo6j2Jqf3DJ+/z9UBKvtuF1brUlv8tOxIDnmdCDcaOLkewifSafn5GjtYGjJNVlqOL5mQsLEYMmWrRT7kxiVWbW08hsW1XT3ZGmhHg3nDOsYFqOGh0ezV7wD5D4/4FfWr+R26E8yXkTZXUJNsgihs6IjvDVreeRsgTe8SpfcEHViE75qddernKIPQywXPLqkUzo4aOug9Eb+2rYgKRQMCj91WDqNTiuWAAuBxrhgpyY5vgI31bZv21dhqYALMH9KrS4ZztN8qfM18wCHhXju5qOqT1mocgZoVyK5piBKcwijPQHhIn0vpjXY70GsvWO8T3gXBSLv1ucpIfnHjLW46LU0oqimkWQnpMYtHtfDAituLiCQsC8VRzZHyrzRAe7bE5DNeByPRjmALJc2HmLT2rp8TR6B9KG3mo9dpx5Fbtm21sohFDaZp2nsWrefuM=
   app: etho-boost
-  run: python3 manage.py db downgrade
```

Comment:
downgrading will constantly wipe our db every time. happy to talk about the challenges here